src/bicep/examples/deploymentVariables.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Please remove this file as we are now excluding it in the `.gitignore`

Reply:
Done.